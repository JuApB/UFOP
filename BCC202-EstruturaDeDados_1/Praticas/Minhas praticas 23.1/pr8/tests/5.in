5
36
Henrique 82 3
Luana 13 3
Henrique 30 1
Miguel 51 2
JoaoGabriel 45 3
Miguel 16 3
Alicia 89 3
Lavinia 77 3
Eduardo 63 2
Otavio 45 1
Enrico 100 1
Pietro 58 2
LucasGabriel 89 1
Danilo 21 3
Benjamin 24 2
Pedro 25 3
Mariah 44 1
Isaac 95 3
Aurora 34 1
Augusto 91 1
Luna 69 2
Yago 29 3
MariaValentina 40 3
MariaHeloisa 72 2
Marina 71 2
MariaLaura 59 2
Elisa 24 2
Tomas 88 1
Victor 90 1
MariaIsis 84 1
Joaquim 53 1
Emanuelly 87 1
Esther 71 2
Allana 94 1
Helena 29 1
DaviMiguel 40 1
40
MariaHeloisa 42 1
Rebeca 47 1
Joaquim 95 3
Caleb 97 3
LuizMiguel 94 2
Bianca 74 1
Julia 24 3
Analu 96 2
AnaSophia 48 1
EnzoMiguel 50 1
MariaJulia 12 3
Gustavo 80 2
Gabriela 30 2
Heitor 99 2
Murilo 73 2
Lais 43 3
Levi 80 2
Andre 10 2
Felipe 53 1
Martin 31 3
Nathan 88 2
Victor 66 3
Isis 20 1
AnaLuiza 55 2
Emanuel 94 3
Manuela 81 3
Eloa 66 1
Luiza 24 3
Mariana 10 3
MariaIsis 79 2
AnaCecilia 86 3
Lorena 32 2
Davi 46 2
Larissa 95 1
Augusto 82 1
ArthurMiguel 37 1
Benjamin 35 3
Olivia 63 3
DaviMiguel 98 1
AnaLiz 96 2
50
Benjamin 35 1
Augusto 12 2
DaviMiguel 46 3
Livia 13 1
Heloisa 32 3
Nathan 97 3
Maitê 65 1
Catarina 99 3
MariaIsis 99 3
Valentina 20 3
Victor 59 2
Leonardo 93 3
Heloise 10 2
Francisco 12 2
JoaoPedro 40 2
Lara 46 3
Liz 84 2
AnaVitoria 88 3
VictorHugo 17 3
Gustavo 46 3
JoaoGabriel 97 2
Daniel 32 1
Malu 96 1
Rodrigo 27 2
Thiago 65 2
Thomas 10 2
Gabriela 63 3
Andre 82 2
Bruno 34 2
LuizOtavio 54 2
Yasmin 45 3
Stella 43 2
Maria 17 1
Caua 19 1
LuizFelipe 24 3
Gabrielly 20 2
Felipe 63 1
MariaEduarda 61 2
Agatha 42 2
Emilly 10 2
ArthurGabriel 18 3
Bryan 33 3
Ruan 75 3
Miguel 63 3
Bruna 76 1
Fernando 60 1
Lucca 96 2
Julia 17 1
Sophia 78 1
MariaFernanda 67 3
36
Lucca 38 2
Gael 36 3
Amanda 59 3
Miguel 78 2
Enzo 46 3
Pietro 26 3
Nicolas 71 1
Nicole 23 3
AnaCecilia 100 3
AnaLaura 18 1
Miguel 70 3
DaviLuiz 94 1
Francisco 100 2
Mirella 18 1
Luna 29 2
Eloa 15 2
Matheus 15 2
Lara 48 1
MariaSophia 49 2
Kaique 74 3
Erick 56 2
MariaHelena 20 3
Emanuel 28 3
Mariah 53 3
Antonia 55 3
Vinicius 12 2
Isadora 45 2
Ana 22 1
Leonardo 80 1
Pedro 38 1
Luana 90 3
Otavio 12 2
Fernanda 31 1
Yuri 75 1
LuizFelipe 60 2
Levi 63 2
30
Cecilia 51 3
EnzoGabriel 75 3
JoaoGabriel 72 1
ArthurHenrique 88 3
Eduarda 70 2
Olivia 28 2
Rafaela 82 3
AnaLiz 84 2
Enzo 57 3
Maitê 52 3
Henry 65 3
DaviLucca 30 2
Luna 97 1
Augusto 34 3
PedroMiguel 51 2
Ana 84 1
JoaoGuilherme 58 1
Andre 34 3
Emanuelly 36 1
MariaJulia 92 1
Lorena 25 3
Vinicius 77 3
Caio 90 3
Felipe 97 3
Murilo 74 1
Gustavo 96 2
JoaoMiguel 42 2
LuizOtavio 65 2
Esther 22 2
Beatriz 65 1
