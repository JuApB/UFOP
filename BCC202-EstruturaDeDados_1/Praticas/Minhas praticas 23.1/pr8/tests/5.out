AnaCecilia 100 3
MariaIsis 99 3
Catarina 99 3
Caleb 97 3
Felipe 97 3
Manuela 81 3
Alicia 89 3
Luana 90 3
Nathan 97 3
Emanuel 94 3
MariaFernanda 67 3
DaviMiguel 46 3
Analu 96 2
Valentina 20 3
Danilo 21 3
Kaique 74 3
Mariah 53 3
Pietro 26 3
Isaac 95 3
Leonardo 93 3
EnzoGabriel 75 3
Gabriela 63 3
Henry 65 3
Heloisa 32 3
Bryan 33 3
Levi 80 2
Liz 84 2
Agatha 42 2
ArthurGabriel 18 3
Benjamin 24 2
Thiago 65 2
Miguel 70 3
Amanda 59 3
Enzo 46 3
LuizFelipe 60 2
Levi 63 2
MariaHelena 20 3
Lavinia 77 3
Joaquim 95 3
AnaVitoria 88 3
Caio 90 3
Enzo 57 3
Miguel 63 3
Lais 43 3
Benjamin 35 3
LuizFelipe 24 3
AnaLiz 96 2
Julia 24 3
Lucca 96 2
Luiza 24 3
Miguel 16 3
Bruno 34 2
MariaLaura 59 2
Andre 10 2
Elisa 24 2
Victor 90 1
Augusto 12 2
Nathan 88 2
JoaoGabriel 97 2
MariaIsis 84 1
Malu 96 1
Sophia 78 1
AnaLuiza 55 2
Luana 13 3
Gael 36 3
Antonia 55 3
Nicole 23 3
Davi 46 2
Emanuel 28 3
Larissa 95 1
Luna 29 2
MariaSophia 49 2
Erick 56 2
Miguel 51 2
Eduardo 63 2
Isadora 45 2
MariaHeloisa 42 1
Victor 66 3
Henrique 82 3
Matheus 15 2
Ruan 75 3
AnaCecilia 86 3
ArthurHenrique 88 3
Lara 46 3
Cecilia 51 3
Olivia 63 3
Gustavo 46 3
Augusto 34 3
MariaValentina 40 3
Andre 34 3
Lorena 25 3
MariaJulia 12 3
Stella 43 2
Gabrielly 20 2
LuizOtavio 65 2
Pietro 58 2
Gustavo 80 2
Victor 59 2
MariaHeloisa 72 2
Rodrigo 27 2
Marina 71 2
Thomas 10 2
Heloise 10 2
LucasGabriel 89 1
Francisco 12 2
Maitê 65 1
JoaoPedro 40 2
Livia 13 1
Maria 17 1
Caua 19 1
DaviMiguel 98 1
EnzoMiguel 50 1
Felipe 63 1
Benjamin 35 1
Tomas 88 1
Henrique 30 1
Emilly 10 2
Isis 20 1
LuizMiguel 94 2
Daniel 32 1
Bruna 76 1
Fernando 60 1
Bianca 74 1
Julia 17 1
Joaquim 53 1
Eloa 66 1
Gabriela 30 2
Emanuelly 87 1
Mariana 10 3
Heitor 99 2
Pedro 25 3
Esther 71 2
MariaIsis 79 2
Murilo 73 2
Miguel 78 2
Mariah 44 1
Lorena 32 2
Allana 94 1
Francisco 100 2
Helena 29 1
Leonardo 80 1
Augusto 82 1
Eloa 15 2
ArthurMiguel 37 1
Rebeca 47 1
AnaLaura 18 1
DaviMiguel 40 1
Felipe 53 1
Lucca 38 2
AnaSophia 48 1
DaviLuiz 94 1
Aurora 34 1
Vinicius 12 2
Ana 22 1
Mirella 18 1
Pedro 38 1
Nicolas 71 1
Otavio 12 2
Fernanda 31 1
Yuri 75 1
Augusto 91 1
Lara 48 1
Martin 31 3
Luna 69 2
Rafaela 82 3
Yasmin 45 3
Vinicius 77 3
Otavio 45 1
Gustavo 96 2
AnaLiz 84 2
VictorHugo 17 3
JoaoGabriel 45 3
Maitê 52 3
DaviLucca 30 2
JoaoMiguel 42 2
Yago 29 3
PedroMiguel 51 2
Ana 84 1
JoaoGuilherme 58 1
Andre 82 2
Emanuelly 36 1
MariaJulia 92 1
LuizOtavio 54 2
Enrico 100 1
Eduarda 70 2
Olivia 28 2
Murilo 74 1
Luna 97 1
JoaoGabriel 72 1
MariaEduarda 61 2
Esther 22 2
Beatriz 65 1
